referrer-policy/generic/subresource/xhr.py

```diff
@@ -0,0 +1,13 @@
+import json
```

Comment:
Nit: I'd have named this file something like `echo-headers.py`. It's equally useful for XHR. and `fetch()`.